src/media/js/site/components/page.js

```diff
@@ -6,15 +6,18 @@ import {reverse, ReverseLink} from 'react-router-reverse';
 
 export class Page extends React.Component {
   static propTypes = {
+    className: React.PropTypes.string,
     title: React.PropTypes.string.isRequired,
     subnav: React.PropTypes.element,
   };
 
   render() {
     return (
-      <section className="page">
+      <section className={classnames(this.props.className, 'page')}>
         <PageHeader {...this.props}/>
-        {this.props.children}
+        <main className="page--main">
```

Comment:
i'd just make it a div. i like having only one main in a document. it's nice that we could do ```document.querySelector('main')```